core/src/main/java/org/apache/gravitino/storage/relational/service/CatalogMetaService.java

```diff
@@ -83,22 +86,15 @@ public CatalogPO getCatalogPOByName(String metalakeName, String catalogName) {
     return catalogPO;
   }
 
+  @Monitored(
```

Comment:
If all the method are using `GRAVITINO_RELATIONAL_STORE_METRIC_NAME+xx`, I would suggest to use two arguments for `@Monitored`